scout-engines/Dockerfile

```diff
@@ -172,9 +121,13 @@ RUN cd wabt-bls12-bignums-fasthost-fastmont-no-superops && make clang-release -j
 #RUN git clone --recursive --single-branch --branch bignum-hostfuncs-bn128-rolluprs https://github.com/ewasm-benchmarking/scout_wabt.cpp.git scoutcpp-bn128-rolluprs
 #RUN cd scoutcpp-bn128-rolluprs && mkdir build && cd build && cmake -DBUILD_TESTS=OFF -DCMAKE_BUILD_TYPE=Release .. && make -j4
 
-
 WORKDIR /scoutyamls
 
+
+# install node 12
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash
```

Comment:
@cdetrio this is actually moved from an earlier place in the docker image, not added as a new. Apparently it was always comparing against node12 in scout-engines.